pkg/operation/botanist/secrets.go

```diff
@@ -475,7 +481,10 @@ func (b *Botanist) CreateNewServiceAccountSecrets(ctx context.Context) error {
 
 			// Wait until we are allowed by the limiter to not overload the kube-apiserver with too many requests.
 			if err := limiter.Wait(ctx); err != nil {
-				return err
+				rateLimitsExceededRegexp := regexp.MustCompile(`(?i)(RequestLimitExceeded|Throttling|Too many requests)`)
+				if rateLimitsExceededRegexp.MatchString(err.Error()) {
+					return gardencorev1beta1helper.NewErrorWithCodes(err, gardencorev1beta1.ErrorInfraRateLimitsExceeded)
+				}
```

Comment:
Remove this - this is an internal limiter and errors here are not related to something users could influence.